Build/scripts/pgo/pogo_training.ps1

```diff
@@ -27,9 +23,18 @@ param (
 
     # force callers to specify this in case of future use
     [Parameter(Mandatory=$True)]
-    [string]$flavor = ""
+    [string]$flavor,
+
+    [string]$vcinstallroot = ${env:ProgramFiles(x86)},
```

Comment:
Okay, we can follow up on desired changes for future iterations. We can talk in person today or Monday about this.